### PR TITLE
feat: update msg to follow new definitions

### PR DESCRIPTION
**Pull request by Julieta11**
New Friend Request feature. Find more info about the proposal [here](https://www.notion.so/decentraland/PRD-New-Friend-Requests-188555225272448f91f3eea5e84f1cd3).

### CODE REVIEW

**Comment by menduz** [comment]
Hi Juli, `status: Final` is meant for implemented ADR that became read-only, kind of a written-in-stone thing. Would you mind changing it to `Review` or `Draft` until fully deployed? In that moment, I recommend to use `Living` instead of `Final` so we can update it as needed

**Review by marianogoldman — CHANGES_REQUESTED** [review]

File: content/ADR-137-new-friend-request.md [hunk 1]
```diff
@@ -26,54 +26,51 @@ The intention of this document is:
...
 sequenceDiagram
   participant renderer
   participant kernel
   
-note left of renderer: the user send a friend request from the pop-up
-renderer->kernel: RequestFriendship(requestFriendshipPayload{messageId:'<unique-id>', userId:'0x...', messageBody='hello!'})
-note right of kernel: ask the server to create the friend request.
-note left of kernel: in case of success
+note left of renderer: The user sends a friend request from the pop-up.
+renderer->kernel: GetFriendRequests(SendFriendRequestPayload{ userId:'0x...', messageBody='hello!' })
```
[the remark this answers is not in the document]
Comment (Julieta11) [reply content/ADR-137-new-friend-request.md:47]: hi @marianogoldman! would you mind telling me specifically where would you add the parentheses? I read your comments and I don't see it hah 🥲

I'm just following what it was like before my changes. I think it follows the following reasoning, does it make sense to you?

image

File: content/ADR-137-new-friend-request.md [hunk 2]
```diff
@@ -178,99 +153,102 @@ sequenceDiagram
...
+RejectFriendRequestReply {
+  oneof message {
+    RejectFriendRequestReplyOk reply = 1;
+    FriendshipErrorCode error = 2;
+  }
 }
 ```
 
-## Get Friend Request list
+## Get Friend Request List
```
[the remark this answers is not in the document]
Comment (Julieta11) [reply content/ADR-137-new-friend-request.md:185]: There was a definition change that led to that message being discarded.

File: content/ADR-137-new-friend-request.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (marianogoldman) [reply content/ADR-137-new-friend-request.md]: Oh, I thought there should be a parenthesis after `SendFriendRequestPayload` and before the `{...}`, but perhaps not needed. I don't even know what the language is and whether the syntax is correct. So please disregard if that is correct.

**Review by marianogoldman — APPROVED** [review]
If the parenthesis things are ok as they are, then 👍

**Pull request merged**